tests/modules/nf-core/somalier/relate/nextflow.config

```diff
@@ -3,7 +3,4 @@ params.prefix = ""
 process {
     publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
 
-    withName:SOMALIER_RELATE {
-        ext.prefix = { params.prefix ? "${params.prefix}." : ""}
```

Comment:
I wonder what the purpose of that line was?

Above `params.prefix` was set to `""`, so `ext.prefix` would also always be set to `""`  🤔

Reply:
Yeah that's the reason I removed it. `params.prefix` isn't even set...

Reply:
Can line 1 also be removed?

Reply:
Welp, overlooked that one :D Thanks!